docs/usage/modularizing.md

```diff
@@ -0,0 +1,77 @@
+
+(modularizing-python-codes)=
+# Modularizing Python codes
+
+Pyodide provides a simple API {any}`pyodide.runPython` to run Python codes.
+However, when your Python codes grow bigger, putting hundreds of lines inside `runPython` is not scalable.
+
+The best way to run Python code with Pyodide is:
+
+1. write a Python package
+1. load your Python package into the Pyodide (Emscripten) virtual file system
+1. import the package with ``let mypkg = pyodide.pyimport("mypkgname")``
+1. call into your package with ``mypkg.some_api(some_args)``.
+
+## How to load Python package into the virtual file system
+
+If the package is built as a `wheel` file, use {any}`micropip.install` to
+install the package.
```

Comment:
Worth noting that you can use `unpackArchive` to install it just as if it were a zip file. We should say that the advantage of micropip is that it does dependency resolution.

Reply:
Added some warning at the bottom of the document